packages/now-cli/src/commands/alias/index.js

```diff
@@ -100,9 +106,11 @@ export default async function main(ctx) {
       '--no-verify': Boolean,
       '--rules': String,
       '--yes': Boolean,
+      '--next': Number,
       '-n': '--no-verify',
       '-r': '--rules',
       '-y': '--yes',
+      '-N': '--next',
```

Comment:
Why does this one have `-N` but `list.js` does not. Let's keep these consistent and remove `-N` for now.

Reply:
@styfle I just created a PR where I add `-N` for `list.js` as well. https://github.com/zeit/now/pull/3948